scripts/install/opt/seagate/cortx/motr/bin/motr_mini_prov.py

```diff
@@ -345,12 +347,21 @@ def update_copy_motr_config_file(self):
     dirs = [MOTR_M0D_DATA_DIR, MOTR_M0D_ADDB_STOB_DIR, MOTR_M0D_TRACE_DIR, MOTR_M0D_CONF_DIR]
     create_dirs(self, dirs)
 
+    # Set ADDB size according to setup_size:
```

Comment:
Do we still need this part, as for large setups motr.conf will be update for this record ?